tests/proof/ple2.fq

```diff
@@ -1,13 +1,13 @@
 fixpoint "--rewrite"
 
 constant maker    : (func(0, [int; QQ]))
-constant QQ       : (func(0, [int; QQ]))
-constant selector : (func(0, [QQ; int]))
 
-match selector QQ x = (x)
+data QQ 0 = [
```

Comment:
editing this file, since it looked wrong: where is no way the `QQ` equation derived from a reflected function, since the LHS is not of the form `f x = ... `.